docs/src/pages/guides/guide-web-components/index.md

```diff
@@ -0,0 +1,111 @@
+---
+id: 'guide-web-components'
```

Comment:
I think this also needs to be added to a yml file to make it appear in the sidebar of the docs site

Reply:
You're right, just updated!